Packs/Core/Playbooks/playbook-Impossible_Traveler.yml

```diff
@@ -777,7 +790,7 @@ tasks:
     message:
       to:
         complex:
-          root: inputs.UserManagerEmail
+          root: UserManagerEmail
```

Comment:
Are you sure we have this value? 
shouldnt it be from the playbook output? 
![image](https://user-images.githubusercontent.com/61933087/205728117-9a004206-15ff-4291-844b-aa4372747c5b.png)